package.json

```diff
@@ -19,7 +19,7 @@
     "THIRD-PARTY-NOTICES"
   ],
   "scripts": {
-    "build": "rm -rf lib/** && tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json && npm run format-check && npm run api-extractor && npm run generate-docs && npm run generate-notices && vite build",
+    "build": "rm -rf lib/** && tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json && npm run prepare && npm run format-check && npm run api-extractor && npm run generate-docs && npm run generate-notices && npx playwright install --with-deps && vite build",
```

Comment:
We should not be installing any depencies in the build

Reply:
done